### DISCUSSION TITLE
Let user set point data type

### DISCUSSION THREAD

**Discussion by NubeDev** [NubeIO/rubix-point-server#210] [General]
Let user set point type

- string
- float
- int

**Comment by RaiBnod** [comment]
It should be only for generic point.

**Comment by RaiBnod** [comment]
@NubeDev while we use the point data type as `string`, we need to change on followings:

On `point_store` and `point_store_history` tables:

- `value` from `float` to `string` (we store `1` as `"1"`, `"value"` as `"value"`, `"20.12"` as `"20.12"`)
- `value_original` `float` to `string`

& also on `InfluxDB` we will store value as `string` previously it was storing as `float`.

Moreover, our `priority_array` array fields values are also `float`, they needs to convert it on `string`.

Please confirm on it before proceeding the changes.

**Comment by Shiny380** [comment]
Related #80 
This is a major change and should be left until later after better discussion
More important issues to be worked on atm

**Comment by RaiBnod** [comment]
After discussion, we are adding `generic_point > type` and if `type=string` we are storing `value_raw` as `string` value leaving value as `null` aside.
